apps/api/src/order/order.service.ts

```diff
@@ -20,20 +20,27 @@ export class OrderService {
     @InjectModel(Order.name) private readonly orderModel: Model<Order>,
     private cartService: CartService,
     private productService: ProductsService,
-    private jwtSerivce: JwtService,
+    private jwtService: JwtService,
     private userService: UsersService,
     private authService: AuthService,
   ) {}
 
-  // Objectives:
-  // Merchant Shipping Fee
-  // Product Matching - for loop userCart
-  // Product Inventory - Update Quantity
-  // Computation for Total Price
-  // Delete Items from Cart
+  async findOne(id: any): Promise<Order> {
+    const order = await this.orderModel.findOne({ 'auth._id': id });
+
+    if (!order) {
+      throw new NotFoundException('Order is not found');
+    }
+
+    return order;
+  }
+
+  async findAllOrders(id: any): Promise<Order[]> {
```

Comment:
better to name it as 'findAll' only since this service is already exclusive for orders.

pls note for future reference @5andrex @TreshAnn @markjoshuacarmen

Reply:
othen than that, LGTM